synapse/lib/storm.py

```diff
@@ -1414,3 +1415,65 @@ def _stormOperTask(self, query, oper):
         for tname in args:
             evt = ':'.join(['task', tname])
             [core.fire(evt, node=node, storm=True, **opts) for node in nodes]
+
+    def _stormOperTree(self, query, oper):
+
+        args = oper[1].get('args')
+        opts = dict(oper[1].get('kwlist'))
+
+        # Prevent bad recursions????
+        recurlim = opts.get('recurlim', 20)
+
+        srcp = None
+        dstp = args[0]
```

Comment:
What if there are no args?

Reply:
Added a bad syntax error in the event of empty args.